### PR TITLE
Fix/genesis autoscaling

### PR DESCRIPTION
**Pull request by Despire**
Closes  https://github.com/berops/claudie/issues/1522

### CODE REVIEW

File: services/autoscaler-adapter/node_manager/utils.go [hunk 1]
```diff
@@ -10,6 +12,163 @@ import (
...
+		"vcpu-12_memory-36g_disk-80g_nvidia1080ti-3",
+		"vcpu-16_memory-48g_disk-80g_nvidia1080ti-4",
+		"vcpu-20_memory-60g_disk-80g_nvidia1080ti-5",
+		"vcpu-24_memory-72g_disk-80g_nvidia1080ti-6",
+		"vcpu-28_memory-84g_disk-80g_nvidia1080ti-7",
+		"vcpu-32_memory-96g_disk-80g_nvidia1080ti-8",
+	}
+
+	m := make(map[string]*typeInfo, len(instanceTypes))
+	for _, typ := range instanceTypes {
```
Comment (JKBGIT1) [inline services/autoscaler-adapter/node_manager/utils.go:151]:
```suggestion
	for _, type := range instanceTypes {
```

for some reason `typ` looks strange to me

File: services/autoscaler-adapter/node_manager/utils.go [hunk 2]
```diff
@@ -10,6 +12,163 @@ import (
...
+		serverName := typ
```
Comment (JKBGIT1) [inline services/autoscaler-adapter/node_manager/utils.go:152]:
```suggestion
		serverName := type
```

has to be applied when the previous does

**Review by JKBGIT1 — APPROVED** [review]
Left some small suggestions. Otherwise LGTM :+1:

File: services/autoscaler-adapter/node_manager/utils.go (hunk 2 above)
Comment (Despire) [reply services/autoscaler-adapter/node_manager/utils.go:152]: type is a Go keyword.

**Pull request merged**